### ISSUE TITLE
error E0446: private type `__QueryWrapper` in public interface

### ISSUE THREAD

**Issue by nerditation** [pandaman64/serde-query#7]
this crate is great for parse deep nested data models, but I can't figure out how the  macro expansion would leak private type in public interface.

if I define the struct as private type, it is all good, but I want the type be public, which just give me error:

```rust
	#[derive(DeserializeQuery)]
	pub struct DeviceRoot {
		#[query(".device.friendlyName")]
		pub device_name: String,
	}
```

```console
error[E0446]: private type `__QueryWrapper` in public interface
  --> src\main.rs:87:11
   |
87 |     #[derive(DeserializeQuery)]
   |              ^^^^^^^^^^^^^^^^
   |              |
   |              can't leak private type
   |              `__QueryWrapper` declared as private
   |
   = note: this error originates in the derive macro `DeserializeQuery` (in Nightly builds, run with -Z macro-backtrace for more info)

```

any idea why would this happen and how to fix it? thanks.

**Comment by pandaman64** [comment]
@nerditation Hi. I released `0.1.5` with the fix. Can you try this?

**Comment by nerditation** [comment]
it works great. thanks for the quick response.
